src/c_src/build-emscripten.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
```

Comment:
I added my build script that converts source into quiet-emscripten.js/quiet-emscripten.js.mem. I think it's appropriate to ship the compiled emscripten stuff on npm, but the github repo ought to include the proper build tools for people who don't want the provided blobs, IMO